content/coronavirus_landing_page.yml

```diff
@@ -206,9 +206,9 @@ content:
       to_watch: "to watch the live stream"
       or: "or"
       watch_link_text: "Watch the live stream on YouTube"
-    date: 2nd April 2020
+    date: 3rd April 2020
```

Comment:
The days are starting to blend into each other - I think we should start putting comments with what day of the week it is. 😆